packages/ui/Cancelable/Cancelable.js

```diff
@@ -36,7 +39,7 @@ const defaultConfig = {
 	onCancel: null,
 
 	/**
-	 * When `true`, the Cancelable instance will handle cancel events globally that successfully
+	 * When `true`, the `Cancelable` instance will handle cancel events globally that successfully
```

Comment:
We don't use the "When true..." text.  We now assume the text is written for the true case.